scripts/run-android-tests.ps1

```diff
@@ -0,0 +1,29 @@
+Set-StrictMode -Version latest
+$ErrorActionPreference = "Stop"
+
+Push-Location $PSScriptRoot/..
+try
+{
+    if (!(Get-Command xharness -ErrorAction SilentlyContinue))
+    {
+        dotnet tool install Microsoft.DotNet.XHarness.CLI --global --version "1.*-*"
+    }
+
+    $tfm = 'net7.0-android'
```

Comment:
do we need to update this if we update the TFM we support to net8?

Reply:
this is a script to run the test locally. It shows as a new file because I wasn't going to update separate .cmd and .sh so I've just merged them to a .ps1 script. In the end, I've had to roll back the change here back to net7 because of test failures, just to unblock this PR (and local builds that were broken) and in another PR, I want to update the device-test apps to run net8